lib/elixir/lib/task.ex

```diff
@@ -293,6 +293,19 @@ defmodule Task do
   """
   @opaque ref :: reference()
 
+  @typedoc """
+  A keyword list containing options for `async_stream` and `async_stream_nolink`
+  functions from the `Task` and `Task.Supervisor` modules.
+  """
+  @typedoc since: "1.17.0"
+  @type async_stream_options :: [
```

Comment:
Usually we tend to go with a type for the single option, like `async_stream_option() :: {:ordered, boolean} | ...` and then do `[async_stream_option()]` in the spec. See `GenServer.option/0` for example

Reply:
Yes, sorry I pushed too fast.
I also realized that there is a difference between `Task` (no `:shutdown`) and `Task.Supervisor` (has `:shutdown`), will clean this up.